dnsmenace.py

```
@@ -340,19 +417,734 @@ def query(
+                        )
+                        record_count += 1
+                        if record_count >= 50:  # Limit display
+                            break
+                    if record_count >= 50:
+                        break
+                if record_count >= 50:
+                    break
+
+            console.print(table)
+            if record_count >= 50:
+                console.print(f"[dim]... and more records (showing first 50)[/dim]")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)